### PR TITLE
Improve README and tutorial

### PR DESCRIPTION
**Pull request by paulmenzel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,17 +7,17 @@ libraries that have interesting bugs, hard-to-find code paths, or other
...
 # See also
 
 * [AddressSanitizer](http://clang.llvm.org/docs/AddressSanitizer.html)
 
 # Contributing
-See [CONTRIBUTING](CONTRIBUTING) first. 
-If you want to add one more benchmark to the test suite,
-simply mimic one of the existing benchmarks and send the pull request. 
+See [CONTRIBUTING](CONTRIBUTING) first.
+If you want to add one more benchmarks to the test suite,
```
Comment (morehouse) [inline README.md:19]: Either "add more benchmarks" or "add another benchmark" sounds better.

**Review by morehouse — CHANGES_REQUESTED** [review]
Looks good besides the one grammatical issue below.

File: README.md (hunk 1 above)
Comment (paulmenzel) [reply README.md:19]: Indeed, I missed the *one* beforehand. I improved the wording as you suggested.

File: tutorial/libFuzzerTutorial.md [hunk 2]
```diff
@@ -5,13 +5,13 @@ In this tutorial you will learn how to use [libFuzzer](http://libfuzzer.info)
...
 
 You will also learn basics of
-[AddressSanitizer](http://clang.llvm.org/docs/AddressSanitizer.html) --
+[AddressSanitizer](https://clang.llvm.org/docs/AddressSanitizer.html) --
 a dynamic memory error detector for C/C++.
 
 Prerequisites: experience with C/C++ and Unix shell.
 
 
-## Setup the environment
```
Comment (kcc) [inline tutorial/libFuzzerTutorial.md:14]: It's a reasobale chance, but it will invalidate the link to the subsection, so maybe not...

File: tutorial/libFuzzerTutorial.md [hunk 3]
```diff
@@ -81,7 +81,7 @@ Now try running it:
 You will see something like this:
 ```
 INFO: Seed: 3918206239
-INFO: Loaded 1 modules (14 guards): [0x73be00, 0x73be38),
```
Comment (kcc) [inline tutorial/libFuzzerTutorial.md:84]: Please avoid space-only changes  -- they complicate the review. 
If you do want to remove trailing spaces -- please do it in a separate PR.

**Comment by dietcookies** [comment]
👌

**Comment by morehouse** [comment]
Tutorial has been moved to https://github.com/google/fuzzing
